mod/cp_notifications/views/default/plugins/cp_notifications/usersettings.php

```diff
@@ -1,410 +1,362 @@
+	$chk_email_grp = create_checkboxes($user->getGUID(), "cpn_email_{$group->getGUID()}", array("sub_{$group->getGUID()}", "set_notify_off"), elgg_echo('label:email'));
+	$chk_site_grp = create_checkboxes($user->getGUID(), "cpn_site_mail_{$group->getGUID()}", array("'sub_site_{$group->getGUID()}", "set_notify_off"), elgg_echo('label:site'));
 
 
-?>
+    //$content .= "		<div class='list-break clearfix'>";
+	$content .= "			<div class='namefield col-sm-8'> <strong> <a href='{$group->getURL()}' id='group-{$group->guid}'>{$group->name}</a> </strong> </div>";
+    $content .= "			<div class='col-sm-2'>{$chk_email_grp}</div>";
+    $content .= "			<div class='col-sm-2'>{$chk_site_grp}</div>";
+	//$content .= "		</div>";
 
-<script>
-	function create_group_content_item(grp_guid, usr_guid) {
-		
-		if (!$('#group-content-' + grp_guid).is(':visible')) {
-			
-			// // assuming this is doing what i think it is doing + loading indicator
-			$('#group-content-' + grp_guid).children().remove();
-			$('#group-content-' + grp_guid).append("<div class='clearfix col-sm-12 list-break'>" + elgg.echo('cp_notify:setting:loading') + "<div>");
 
-			// jquery - retrieve the group content on user click (this way it doesn't try and load everything at once)
-			elgg.action('cp_notify/retrieve_group_contents', {
-				data: {
-					group_guid: grp_guid,
-					user_guid: usr_guid,
-				},
+	// GROUP CONTENT SUBSCRIPTIONS
+    $content .= '		<div class="accordion col-sm-12 clearfix mrgn-bttm-sm">';
+	$content .= '			<details class="acc-group" onClick="return create_group_content_item('.$group->getGUID().', '.$user->getGUID().')">';
+	$content .= "				<summary id='group_item_container-{$group->getGUID()}' class='wb-toggle tgl-tab'>".elgg_echo('cp_notify:groupContent').'</summary>';
+    $content .= "				<div id='group-content-{$group->getGUID()}' class='tgl-panel clearfix'></div>";
+    $content .= '			</details>';	
+    $content .= '		</div>';			
+   				
+} 
+$content .= '	</div>';
+$content .= "</section>";
 
-				success: function (content_arr) {
-					$('#group-content-' + grp_guid).children().remove();
 
-					// we want to display a message when there arent any items subscribed under a group
-					if (content_arr.output.num_content <= 0) {
-						$('#group-content-' + grp_guid).append("<div class='clearfix col-sm-12 list-break'> " + elgg.echo('cp_notify:setting:no_grp_subscription') +" <div>");
-						return;
-					} 
 
+/// PERSONAL SUBSCRIPTIONS (DISPLAYS ALL ITEMS THAT DO NOT BELONG IN GROUP NOTIFICATIONS)
 
-					// create a list of all the content in the group that you are subscribed to
-					for (var item in content_arr.output.text3)
-						$('#group-content-' + grp_guid).append("<div class='clearfix col-sm-12 list-break'>" + content_arr.output.text3[item] + "<div>");
+// cyu - patched issue with personal subscription that contains group content
+$query = "
+SELECT e.guid, e.subtype as entity_subtype, es.subtype, o.title, o.description
```

Comment:
Have you tried running this query on a db like pre-prod? It doesn't look very large-database friendly...